packages/react/spec/auth/useAuth.spec.ts

```diff
@@ -49,4 +49,14 @@ describe("useAuth", () => {
       rerender();
     }).toThrowErrorMatchingInlineSnapshot(`"[Network] Network error"`);
   });
+
+  test("it throws when the api client does not have a User model", async () => {
+    expect(() => {
+      const { rerender } = renderHook(() => useAuth(), { wrapper: TestWrapper(noUserModelApi) });
+
+      mockNetworkError();
+
+      rerender();
+    }).toThrowErrorMatchingInlineSnapshot(`"[Network] Network error"`);
```

Comment:
Why does this test assert a network error instead of a missing user model error?